frontend/packages/kubevirt-plugin/src/models/index.ts

```diff
@@ -244,3 +244,16 @@ export const NetworkAddonsConfigModel: K8sKind = {
   id: 'networkaddonsconfig',
   crd: true,
 };
+
+export const DataSourceModel: K8sKind = {
+  label: 'DataSource',
+  labelPlural: 'DataSources',
+  apiVersion: 'v1bete1',
+  apiGroup: 'cdi.kubevirt.io',
+  plural: 'datasources',
+  abbr: 'datasources',
```

Comment:
shouldn't this be `ds`?

Reply:
i asked what is the abbr for tat, the answer that I got is none.. so I don't really know. I'm guessing I change it to ds

Reply:
abbr is an abbriviation for abbrivation :) so I think changing to `DS` would be best

Reply:
To find the apprv:
`oc describe crd datasources` on a cluster that has it:
and look for "Short names:"

For example:
```
oc describe crd datavolume
...
Plural:     datavolumes
    Short Names:
      dv
      dvs
    Singular:  datavolume
```
It looks to me like ds does not have abbrv in this api version.

Reply:
@yaacov so should I remove the abbr or leave it as it is?